XTS/sar/resources/xts-properties.xml

```diff
@@ -211,6 +211,7 @@
     <entry key="org.jboss.jbossts.xts.initialisation.xtsInitialisation_1">org.jboss.jbossts.xts.initialisation.CoordinatorSideInitialisation</entry>
     <entry key="org.jboss.jbossts.xts.initialisation.xtsInitialisation_2">org.jboss.jbossts.xts.initialisation.ParticipantSideInitialisation</entry>
     <entry key="org.jboss.jbossts.xts.initialisation.xtsInitialisation_3">org.jboss.jbossts.xts.initialisation.ClientSideInitialisation</entry>
+    <entry key="org.jboss.jbossts.xts.initialisation.xtsInitialisation_4">org.jboss.jbossts.xts.initialisation.RecoveryInitialisation</entry>
```

Comment:
It is so that compensations can load the recovery aspects of the coordinator and participant startups alone (the initialializers normally initialize recovery and endpoints).